requirements_dev.txt

```diff
@@ -1,2 +1,6 @@
 pre-commit~=2.16.0
-taskipy>=1.9.0
+taskipy~=1.9.0
+mypy~=0.910
```

Comment:
mypy as well? I don't see a mypy configuration anywhere.

Reply:
The dependency was added just for the mypy plugin, since pyright checks that one as well